game/languages/Czech.ini

```diff
@@ -572,21 +574,129 @@ PARTY_MODE_TOURNAMENT_ROUNDS_DESC=další hráč k mikrofonu
+;TODO: EDIT_INFO_ALL_TONES_DECREASED_BY_OCTAVE=All notes decreased by an octave
+;TODO: EDIT_INFO_FIX_TIMINGS=Timings corrected
+;TODO: EDIT_INFO_JUMPTO_PREVIEW_AND_PLAY=Jump to preview and play
+;TODO: EDIT_INFO_NOTE_SHORTENED_AT_START=Note shortened at start
+;TODO: EDIT_INFO_NOTE_LENGTHENED_AT_START=Note lengthened at start
+;TODO: EDIT_INFO_NOTE_SHORTENED_AT_END=Note shortened at end
+;TODO: EDIT_INFO_NOTE_LENGTHENED_AT_END=Note lengthened at end
+;TODO: EDIT_INFO_NOTE_SHIFTED_LEFT=Note shifted left
+;TODO: EDIT_INFO_NOTE_SHIFTED_RIGHT=Note shifted right
+;TODO: EDIT_INFO_NOTES_SHIFTED_LEFT=All notes shifted left (from current note)
+;TODO: EDIT_INFO_NOTES_SHIFTED_RIGHT=All notes shifted right (from current note)
+;TODO: EDIT_INFO_NOTE_PITCH_DECREASED=Note pitch decreased
+;TODO: EDIT_INFO_NOTE_PITCH_INCREASED=Note pitch increased
+;TODO: EDIT_INFO_NOTE_SET_TO_NORMAL=Note set to normal
+;TODO: EDIT_INFO_NOTE_SET_TO_GOLDEN=Note set to golden
+;TODO: EDIT_INFO_NOTE_SET_TO_FREESTYLE=Note set to freestyle
+;TODO: EDIT_INFO_NOTE_SET_TO_RAP=Note set to rap
+;TODO: EDIT_INFO_NOTE_SET_TO_RAPGOLDEN=Note set to rap golden
+;TODO: EDIT_INFO_BPM_DECREASED_BY=BPM decreased by
+;TODO: EDIT_INFO_BPM_INCREASED_BY=BPM increased by
+;TODO: EDIT_INFO_GAP_DECREASED_BY=GAP decreased by
+;TODO: EDIT_INFO_GAP_INCREASED_BY=GAP increased by
+;TODO: EDIT_INFO_VIDEOGAP_DECREASED_BY=Videogap decreased by
+;TODO: EDIT_INFO_VIDEOGAP_INCREASED_BY=Videogap increased by
+;TODO: EDIT_INFO_NOTE_DIVIDED=Note divided
+;TODO: EDIT_INFO_CONVERTED_TO_SOLO=Song converted to solo (current track)
+;TODO: EDIT_INFO_CONVERTED_TO_DUET=Song converted to duet
+;TODO: EDIT_INFO_SWITCHED_TO_TRACK=Switched to track
+;TODO: EDIT_INFO_LINE_COPIED_TO_TRACK=Line copied to track
+;TODO: EDIT_INFO_LINE_NOT_COPIED=Line NOT copied (not enough space)
+;TODO: EDIT_INFO_LINE_DIVIDED=Line divided at current note
+;TODO: EDIT_INFO_LINES_JOINED=Joined current and next line
+;TODO: EDIT_INFO_LINE_MOVED_TO_TRACK=Line moved to track
+;TODO: EDIT_INFO_LINE_NOT_MOVED=Line NOT moved (not enough space)
+;TODO: EDIT_INFO_EXIT=Do you want to leave the editor without saving?
+;TODO: EDIT_INFO_CURRENT_LINE=Line:
+;TODO: EDIT_INFO_CURRENT_NOTE=Note:
+;TODO: EDIT_INFO_CURRENT_BEAT=Beat:
+;TODO: EDIT_POPUP_MEDLEY_RELATIVE_UNSUPPORTED=Medley with relative is not supported!
+;TODO: EDIT_POPUP_MEDLEY_DELETED=Medley tags deleted!
+;TODO: EDIT_POPUP_DUET_RELATIVE_UNSUPPORTED=Duet with relative is not supported!
+;TODO: EDIT_POPUP_MEDLEY_DUET_UNSUPPORTED=Medley with Duet is not supported!
+;TODO: EDIT_POPUP_SYNTAX_ERRORS=This song contains some syntax errors!
+;TODO: EDIT_POPUP_NO_MEDLEY_SECTION=No Medley section set. Check your txt file or set it with A (start) and SHIFT+A (end).
+EDIT_NOTENAME_C=C
+EDIT_NOTENAME_C_SHARP=C♯
+EDIT_NOTENAME_D=D
+EDIT_NOTENAME_D_SHARP=D♯
+EDIT_NOTENAME_E=E
+EDIT_NOTENAME_F=F
+EDIT_NOTENAME_F_SHARP=F♯
+EDIT_NOTENAME_G=G
+EDIT_NOTENAME_G_SHARP=G♯
+EDIT_NOTENAME_A=A
+EDIT_NOTENAME_A_SHARP=A♯
```

Comment:
I strictly followed this: https://en.wikipedia.org/wiki/Musical_note
IMHO, if B is used, then also Cis/Des, Dis/Fes, Fis/Ges and so on should be used, shouldn't they? I am not the musical expert though... Fortunately, this can be easily changed/customized in the language files.

Reply:
I'd rather stay consistent, but maybe add 'B' as an alternative: 'A# (B)' (for Czech, German, Danish, Swedish, Finnish, Hungarian, Norwegian, Polish and Slovak, that is).

Reply:
See bd1ebe32e72efdee018a74fc7cdbfd39c64bc977